mantidimaging/gui/windows/live_viewer/view.py

```diff
@@ -33,15 +32,26 @@ def __init__(self, main_window: 'MainWindowView', live_dir_path: Path) -> None:
         self.presenter = LiveViewerWindowPresenter(self, main_window)
         self.live_viewer = LiveViewWidget()
         self.imageLayout.addWidget(self.live_viewer)
-        self.show_first_image()
-
-    def show_first_image(self):
-        """Show the first image in the selected directory."""
-        for file in self.path.iterdir():
-            if file.suffix == ".tif" or file.suffix == ".tiff":
-                self.image = tifffile.imread(file)
-                break
-        self.live_viewer.show_image(self.image)
-
-    def set_image(self, image_data: Optional['np.ndarray'], autoLevels: bool = True):
-        self.spectrum.image.setImage(image_data, autoLevels=autoLevels)
+        self.watch_directory()
+
+    def show_image(self, image: np.ndarray) -> None:
+        """
+        Show the image in the image view.
+        @param image: The image to show
+        """
+        self.live_viewer.show_image(image)
+
+    def watch_directory(self):
+        """Show the most recent image arrived in the selected directory"""
+        self.presenter.set_dataset_path(self.path)
+
+    def show_most_recent_image(self, image_path: np.ndarray) -> None:
+        """
+        Show the most recent image arrived in the selected directory
+        @param image_file_path: The path of the image to show
+        """
+        self.show_image(image_path)
```

Comment:
I think this is not used any more now

Reply:
Thanks for catching this also, same refactoring missed as previous